File: #1011.cpp
// Fly me to the Alpha Centauri
#include <iostream>
#include <cmath>
using namespace std;
int main()
{
    ios::sync_with_stdio(false);
    cin.tie(NULL);
    cout.tie(NULL);

    int t;
    cin >> t;
    
    for(int i = 0; i < t; i++)
    {
        long long x, y;
        cin >> x >> y;
        
        long long diff = y - x;
        long long cnt = 0, max = 0;
        
        while(max * max <= diff) max++;
        
        max--;
        cnt = 2 * max - 1;
        
        long long temp = diff - max * max;
        temp = (long long)ceil((double)temp / (double)max);
        cnt += temp;

        cout << cnt << "\n";
    }
    
    return 0;
}

File: #1011.py
# Fly me to the Alpha Centauri
import math
t = int(input())

for i in range(t):
    x, y = map(int, input().split())
    diff = y - x
    cnt = 0
    max = 0

    while(max * max <= diff):
        max += 1
    
    max -= 1
    cnt = 2 * max - 1

    temp = diff - max * max
    temp = math.ceil(float(temp) / float(max))
    cnt += temp
    
    print(cnt)